globals.go

```diff
@@ -1084,3 +1084,55 @@ const (
 	LogViewerHTML    LogViewer = "html"
 	LogViewerLobster LogViewer = "lobster"
 )
+
+// ContainerOS denotes the operating system of a running container. Only linux and windows are supported.
```

Comment:
I don't think we need to say that it only supports Windows and Linux since it should be obvious from the two consts.